radio/src/translations/se.h

```diff
@@ -31,6 +31,28 @@
  *  \0               -ends current string
  */
 
+
+// New strings from LVGL implementation
```

Comment:
Can you please check the english original and try to have these at the same place? That eases further maintenance ;-)

Reply:
Tested on TX16S and Xlite.

Reply:
Sure, I'll do that as soon as I am able.